### PR TITLE
Update package_rsyslog_installed in RHEL6 to consider both rsyslog and rsyslog7 package

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Update package_rsyslog_installed in RHEL6 to consider both rsyslog and rsyslog7 package
#### Rationale:

- Relates to #2642

### CODE REVIEW

**Comment by redhatrises** [comment]
Is rsyslog or rsyslog7 the default install for RHEL6 latest?

**Comment by ggbecker** [comment]
> Is rsyslog or rsyslog7 the default install for RHEL6 latest?

It's `rsyslog`. Feel free to close if that should be the correct value when default installed package is the one we should aim for.

**Comment by redhatrises** [comment]
> > Is rsyslog or rsyslog7 the default install for RHEL6 latest?
> 
> It's `rsyslog`. Feel free to close if that should be the correct value when default installed package is the one we should aim for.

Either we drop or we update the OVAL to check for either rsyslog or rsyslog7. Your call as RHEL6 is EOL 11/30/2020 (outside of EUS which I doubt we are going to anything at this point.)

**Comment by ggbecker** [comment]
I have created a specific OVAL check for RHEL6 which considers both packages `rsyslog` and `rsyslog7` and the special trick:

```
    backends:
        oval@rhel6: "off"
```
in the rule.yml seems to work just fine.

**Review by yuumasato — APPROVED** [review]
> The downside of this implementation is that I had to drop the `EVR` parameter from the template. So it cannot check for a particular version of the package.

I don't see lack of EVR support as a problem. The original rule didn't require a specific version, and updates are unlikely.

File: linux_os/guide/system/logging/package_rsyslog_installed/rule.yml [hunk 1]
```diff
@@ -38,3 +38,5 @@ template:
     name: package_installed
     vars:
         pkgname: rsyslog
+    backends:
+        oval@rhel6: "off"
```
Comment (yuumasato) [inline linux_os/guide/system/logging/package_rsyslog_installed/rule.yml:42]: The build system already has a preference for product specific implementations (`rhel6.xml` vs `shared.xml`).
So this is not strictly necessar.
But I like it as it makes explicit in the `rule.yml` that `rhel6` is not using the template implementation.

**Pull request merged**